spatial-tracking-explainer.md

```diff
@@ -196,12 +196,63 @@ function teleport(deltaX, deltaY, deltaZ) {
 }
 ```
 
+### Identity reference spaces
+An _identity_ reference space is one which provides no tracking. This type of reference space is used for creating inline experiences with tracking information explicitly disabled, such as a furniture viewer that will use [click-and-drag controls](#click-and-drag-view-controls) to rotate the furniture. It also supports cases where the developer wishes to avoid displaying any type of tracking consent prompt to the user prior while displaying inline content.
+
+This type of reference space is requested with a type of `identity` and returns a basic `XRReferenceSpace`. `XRViewerPose` objects retrieved with this reference space will have a transform that is equal to the reference space's `originOffset`. The pose will only contain a single `XRView`, which must also have a transform equal to the `originOffset`.
```

Comment:
> `XRViewerPose` objects retrieved with this reference space will have a transform that is equal to the reference space's `originOffset`.

Isn't this the reverse of what we defined `originOffset` to do for all other reference spaces?

**For the `"bounded"` space:**
Imagine that the viewer is located at position (2, 2, 2) within the default `"bounded"` reference space.  If I then "offset" the bounded reference space's "origin" by (2, 2, 2), I would expect the viewer to now be at position (0, 0, 0), since I shifted that space's origin over by (2, 2, 2) to now coincide with the viewer.

This means that **applying an `originOffset` of (2, 2, 2)** to the `"bounded"` reference space causes the viewer's pose to **translate by (-2, -2, -2)**.

**For the `"identity"` space:**
However, per this PR, if I apply an `originOffset` of (2, 2, 2) to the `"identity"` reference space, the viewer's position changes from (0, 0, 0) to (2, 2, 2).

This means that **applying an `originOffset` of (2, 2, 2)** to the `"identity"` reference space causes the viewer's pose to **translate by (2, 2, 2)**, the opposite behavior as with the `"bounded"` reference space.